researcher_ids/README.md

```diff
@@ -0,0 +1,90 @@
+# Researcher Identity & Access Claims
+
+## Quick Links
+
+- [GA4GH Researcher Identity & Access Claims Specification](http://bit.ly/ri-claims)
+- [GA4GH Authentication and Authorization Infrastructure (AAI) OpenID Connect Profile](http://bit.ly/g4gh-aai)
+
+## Introduction
+
+Research groups around the world are making datasets available to other researchers through data sharing repositories such as EGA and dbGaP. However, access processes are fraught with difficulties and the following shortcomings have been identified:
+
+- It is inconsistent: different Data Access Committees (DACs) may reach different conclusions regarding appropriate access.
+- It is cumbersome:  DACs have a difficult time identifying who a bona fide researcher is, and have difficulty validating their identity.
+- It is not scalable: the number of requests for access to human subjects data is growing at a supralinear rate, and it is growing increasingly difficult for DACs to keep up.
+
+The genomics community was therefore in need of:
+
+- a consistent definition of who a bona fide researcher is
+- one or more identity providers that respect this definition and provide secure digital identities that researchers can use online to access in the virtual world that travel with the researcher across various federated data sharing repositories around the world
```

Comment:
Remove: "in the virtual world that travel with the researcher across"
Remove: "sharing" (after the words "federated data")